system-int-test/src/cmr/system_int_test/utils/ingest_util.clj

```diff
@@ -23,7 +23,8 @@
    [cmr.system-int-test.utils.dev-system-util :as dev-sys-util]
    [cmr.common-app.test.side-api :as side]
    [cmr.ingest.config :as icfg])
-  (:import [java.lang.NumberFormatException]))
+  (:import
+   [java.lang.NumberFormatException]))
```

Comment:
FYI - I think we agreed even for single imports to use the syntax:

`(java.lang NumberFormatException)`